database/factories/BookingFactory.php

```diff
@@ -8,11 +8,13 @@
 $factory->define(Booking::class, function (Faker $faker) {
     $from = $faker->dateTimeThisYear;
     $to = $faker->dateTimeInInterval($from, '+1 day');
+    $network_type = 0;
 
     return [
         'user_id' => $faker->numberBetween(1000000, 9999999),
         'position_id' => factory(\App\Modules\Position\Position::class),
         'from' => $from,
         'to' => $to,
+        'network_type' => $network_type,
```

Comment:
You can inline this aswell (below assumes the using the constants suggested above)
```suggestion
        'network_type' => Booking::NETWORK_TYPE_LIVE,
```